### DISCUSSION TITLE
How can I use several kittens at once?

### DISCUSSION THREAD

**Discussion by dydyamotya** [kovidgoyal/kitty#4093] [Q&A] [answered]
Hello,

To connect to my server I need to use both **ssh** and **clipboard** kittens. But I haven't found a way in docs to use a number of kittens at once. Is it possible?

Thanks for help.

**Comment by kovidgoyal** [comment, marked as answer]
A kitten is just a program. You can use as many of them as you like,
just as you can run as many programs as you like.

But note that after sshing into a server, kitty must be installed on
that server for the kittens to be available.
